CONTRIBUTING.md

```diff
@@ -0,0 +1,46 @@
+# Contributing to ADF 
+
+## Submitting a Pull Request (PR)
+
+Before you submit please follow the following step: 
+
+* Search in  [GitHub PR](https://github.com/Alfresco/alfresco-ng2-components/pulls) for an open or closed PR
+  that could solve or already solve your issue.
+* Search in the [GitHub release history](https://github.com/Alfresco/alfresco-ng2-components/releases) if your
+  issue has already been solved in a new version of ADF.
+* Fork our repository (if you don't know how to do it read this GitHub document)[https://help.github.com/articles/creating-a-pull-request-from-a-fork/]
+* Make your changes in a new git branch starting from development and following the [naming convention](https://github.com/Alfresco/alfresco-ng2-components/wiki/Branching-Strategy)
+
+     ```shell
+     git checkout -b dev-{developerName}-{GitIssuedId/JiraIssueId} development
```

Comment:
I think they can use whatever naming convention they want with their forks. As all of those branches will be private for everyone until a PR is created.